gulpfile.js

```diff
@@ -344,19 +344,19 @@ function checkTypes() {
     checkTypes: true,
     preventRemoveAndMakeDir: true,
   });
-  buildSw({
+  // Temporarily turned off due to unknown type warnings.
+  /*buildSw({
```

Comment:
i tested this over the weekend and it seemed to work.

edit: looks like you already uncommented this https://github.com/ampproject/amphtml/pull/4926/files